File: .claude/plugins/openai-agent-builder/references/project-templates/python-multi-agent/specialists/specialist_2.py
"""{{PROJECT_NAME}} — Specialist agent 2."""
from agents import Agent
from tools import tools

specialist_2 = Agent(
    name="{{SPECIALIST_2_NAME}}",
    instructions="{{SPECIALIST_2_INSTRUCTIONS}}",
    handoff_description="{{SPECIALIST_2_HANDOFF_DESCRIPTION}}",
    tools=tools,
    model="gpt-4.1",
)

File: .claude/plugins/openai-agent-builder/references/project-templates/typescript-multi-agent/src/specialists/specialist2.ts
import { Agent } from '@openai/agents';
import { tools } from '../tools';

export const specialist2 = new Agent({
  name: '{{SPECIALIST_2_NAME}}',
  instructions: '{{SPECIALIST_2_INSTRUCTIONS}}',
  tools,
  model: 'gpt-4.1',
});
